### PR TITLE
added groups creation files for internationalization

### PR DESCRIPTION
**Pull request by martinGerez**
NOTE: This is a needed feature for my application.

Maybe it could change in the future in order to keep simple and maintainable.

Notes:
* I decided to maintain German and add English and Spanish translation for groups name in Groups Creator service.
* Specs are passing.

### CODE REVIEW

File: lib/keepr/groups_creator.rb [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 class Keepr::GroupsCreator
-  def initialize(target)
+  def initialize(target, language)
```
Comment (ledermann) [inline lib/keepr/groups_creator.rb:4]: To maintain backwards compatibility, I prefer a to use this:

`def initialize(target, language = :de)`

File: spec/keepr/groups_creator_spec.rb [hunk 2]
```diff
@@ -3,18 +3,18 @@
 require 'spec_helper'
 
 describe Keepr::GroupsCreator do
-  context 'balance groups' do
+  context 'balance groups in german' do
     before :each do
-      Keepr::GroupsCreator.new(:balance).run
+      Keepr::GroupsCreator.new(:balance, :de).run
```
Comment (ledermann) [inline spec/keepr/groups_creator_spec.rb:8]: This change can then be removed...

File: spec/keepr/groups_creator_spec.rb [hunk 3]
```diff
@@ -24,21 +24,21 @@
 
   context 'profit & loss groups' do
     before :each do
-      Keepr::GroupsCreator.new(:profit_and_loss).run
+      Keepr::GroupsCreator.new(:profit_and_loss, :de).run
```
Comment (ledermann) [inline spec/keepr/groups_creator_spec.rb:27]: ... and this ...

**Review by ledermann — CHANGES_REQUESTED** [review]
Great work, thanks a lot!

There is just a little thing to keep backwards compatibility. If you can change this, I will merge your PR.

File: lib/keepr/groups_creator.rb (hunk 1 above)
Comment (martinGerez) [reply lib/keepr/groups_creator.rb]: Good point, I thought about it after open PR 👌

**Review by ledermann — APPROVED** [review]
👍

**Pull request merged**